### ISSUE TITLE
Couldn't open wallet: device not found: Trezor

### ISSUE THREAD

**Issue by StevenRispoli** [monero-project/monero-gui#4273]
I can no longer connect to the Monero-gui wallet with my Trezor Model T.

Issue title is the exact error message: "Couldn't open wallet: device not found: Trezor"

Monero-gui v0.18.3.1 - flatpak

Trezor Bridge v2.0.27

Pop_OS 22.04

I don't see trezord in system monitor or htop, but it does otherwise appear to be running since I could verify the bridge status and that it could see my Model T connected here: http://127.0.0.1:21325/status/

I've reinstalled the bridge and restarted my PC to no avail.

Monero-gui is the only third-party wallet I use so I have not confirmed if I can connect to other third-party wallets, but I can connect to the desktop and web versions of trezor suite with no issues (though that might not tell us much since trezor suite supposedly has it's own bridge it falls back to).

Is this an issue anyone else has encountered? I'm not sure if it's a Monero-gui issue or a Trezor bridge issue.

Thanks

**Comment by selsta** [comment]
Can you try the GUI binary from getmonero.org? It's possible that the flatpak version was compiled without Trezor support.

**Comment by StevenRispoli** [comment]
That was indeed the problem, thank you. Any reason the flatpak version wouldn't be compiled with Trezor support? Should we refrain from using the flatpak version?

**Comment by selsta** [comment]
@BigmenPixel0 any idea if this changed recently or did flatpak always miss Trezor support?

**Comment by StevenRispoli** [comment]
I've been using the flatpak version with the same Model T for at least a couple of years.

**Comment by andreashuber69** [comment]
> I've been using the flatpak version with the same Model T for at least a couple of years.

Same here, Trezor has been working flawlessly with the flatpak version. Only the latest version fails to detect it.

@Monero Team: Please restore Trezor support in the flatpak version.

(When I try to add a new wallet with hardware support, then Trezor is still offered as an option in the UI.)

**Comment by selsta** [comment]
Some dependencies got updated and it seems that caused issues with the Trezor subsystem, I will try to get it reverted.

https://github.com/flathub/org.getmonero.Monero/commit/9811de679ef12e60f3644197a7b429724689d7f3

**Comment by BigmenPixel0** [comment]
@selsta, sorry, I didn't see past messages in time. Should I revert the update?

**Comment by BigmenPixel0** [comment]
And which module is a problem, if you know?

**Comment by selsta** [comment]
@BigmenPixel0 I would need to see build logs, easiest for now is to revert everything. I'd guess it's related to the protobuf update.

**Comment by BigmenPixel0** [comment]
@selsta you can see them here
https://buildbot.flathub.org/#/builders/6/builds/97527

**Comment by BigmenPixel0** [comment]
@andreashuber69 @StevenRispoli 
Can you try it and tell me if it works?
https://github.com/flathub/org.getmonero.Monero/pull/104#issuecomment-1937001039

**Comment by andreashuber69** [comment]
> @andreashuber69 @StevenRispoli Can you try it and tell me if it works? [flathub/org.getmonero.Monero#104 (comment)](https://github.com/flathub/org.getmonero.Monero/pull/104#issuecomment-1937001039)

Ran the command line in the linked post and now everything is back to where it was, thank you!

**Comment by BigmenPixel0** [comment]
@andreashuber69 
Great. Now you should remove this version and wait for an update (because that's a test package which won't get updates)

**Comment by BigmenPixel0** [comment]
@selsta does the new release support the newest version of protobuf?

**Comment by selsta** [comment]
@BigmenPixel0 no, the newest version of protobuf requires C++17 which we currently don't use on the release branch.
